WebRoot/css/zm.css

```diff
@@ -4021,23 +4021,69 @@ HTML>BODY .appt-selected .appt_allday_body
 }
 
 /* ZmChooseFolderDialog and ZmPickTagDialog */
+
+#ChooseFolderDialog {
+	@ChooseFolderDialog@
+}
+
+#ChooseFolderDialog .WindowOuterContainer {
+	@ChooseFolderDialogContainer@
+}
+
 .ChooserDialog {
-	width:320px;
+	@ChooseFolderDialogBody@
```

Comment:
The property name should not be specific to "Folder" that's what I am referring to because that kind of misleads as it applies for PickTag, ChooseAccount (ZmChooseAccountDialog)  and maybe others. Maybe we can use generic name just ChooserDialogBody etc?

@silentsakky  @kalpeshhpatel  : Your thoughts?